tokio/src/sync/semaphore.rs

```diff
@@ -47,7 +47,23 @@ use std::sync::Arc;
 /// }
 /// ```
 ///
-/// Use [`Semaphore::acquire_owned`] to move permits across tasks:
+/// Limit number of incoming requests being handled at the same time, using a task-safe semaphore
+/// via [`Semaphore::acquire_owned`].
```

Comment:
Let's drop the `acquire_owned` part from the first line, and instead explain its use in the text.
```suggestion
/// Limit number of incoming requests being handled at the same time.
```